buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle

```diff
@@ -126,31 +132,40 @@ class ExampleRunnerTask extends JavaExec {
     }
     catch (Exception e) {
       logQuietMessage('\n\033[0;31mrunExample exception!\033[0m Did ' +
-        'you provide a valid example identifier? E.g. ' +
-        '\033[0;35m--example="basicoperations.GetCampaigns"\033[0m\n\n' +
+        'you provide a valid example identifier? ' + exampleInvocation +
         e.message)
     }
   }
 
+  // Prints a message to the console, even in quiet mode.
   private void logQuietMessage(String message) {
     logger.quiet(message)
   }
 }
 
 class ListExamplesTask extends DefaultTask {
 
-  // The base directory containing all examples.
-  private final String examplesDirectory =
-    'google-ads-examples/src/main/java/com/google/ads/googleads/examples/'
-
-  // Regex for extracting descriptions.
+  // Regex for extracting descriptions. Isolates the Javadoc comment immediately
+  // preceeding the class definition.
   @Input
-  private final String regex =
+  private static final String COMMENT_REGEX =
     '(?s)(/\\*\\*(?:(?!/\\*\\*).)*?\\*/)(?:(?:(?!\\*/).)*?)(?=public class )'
 
   // Regex for comment markers.
   @Input
-  private final String commentMarkers = '(/\\*\\*\\s+|\\*\\s|\\*/)'
+  private static final String COMMENT_MARKERS = '(/\\*\\*\\s+|\\*\\s|\\*/)'
+
+  // The base directory containing all examples.
+  private static final String MAIN_EXAMPLES_BASE_DIRECTORY =
```

Comment:
I just happened to notice that `listExamples` fails if I run it from a subdir.

```
cd google-ads-examples
../gradlew listExamples

Execution failed for task ':google-ads-examples:listExamples'.
> java.io.FileNotFoundException: /usr/local/google/home/<USER>/.gradle/daemon/6.7.1/google-ads-examples/src/main/java/com/google/ads/googleads/examples
```

This doesn't seem to be an issue with other tasks like `build`.

Is it difficult to make this work?

Reply:
Hmm... probably has to do with it using the current directory rather than the project directory. Will investigate.

Reply:
Fixed; should now be current directory agnostic.

Reply:
Thanks. I confirmed this works for me locally as well.